src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -268,6 +272,46 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
   }
   */
 
+  @Override
+  public ResponseEntity<Void> grantRole(
+      @PathVariable("id") UUID id,
+      @PathVariable("role") IamRole role,
+      @PathVariable("memberEmail") String memberEmail) {
+    samService.grantWorkspaceRole(
+        id,
+        getAuthenticatedInfo(),
+        bio.terra.workspace.service.iam.model.IamRole.fromApiModel(role),
```

Comment:
We can't import both the generated `IamRole` and the internal `IamRole`, so one ends up fully specified. That felt fine to me since the Controller is explicitly the only place that will see both, but I could also change one or both names. Thoguhts?

Reply:
Got it. I didn't notice that.